src/api/AdditionalServicesApi.ts

```diff
@@ -296,11 +306,16 @@ export class AdditionalServicesApiImpl implements AdditionalServicesApi {
     }
   }
 
-  public async getMostInterestingAuctionDetails(): Promise<Maybe<AuctionInfo[]>> {
+  public async getMostInterestingAuctionDetails(
+    closedAuctions = false,
+  ): Promise<Maybe<AuctionInfo[]>> {
     try {
       const promises: Promise<Response>[] = []
       for (const networkId in this.urlsByNetwork) {
-        const url = await this.getMostInterestingAuctionDetailsUrl({
+        const fn = closedAuctions
+          ? this.getMostInterestingClosedAuctionDetailsUrl
+          : this.getMostInterestingAuctionDetailsUrl
+        const url = fn.bind(this)({
```

Comment:
I liked, what happened to the `await` sentence?

Reply:
the functions weren't async so it wasn't necessary